docs/drone.adoc

```diff
@@ -0,0 +1,110 @@
+== Cube Docker Drone Integration
+
+=== Drone
+
+The Arquillian Drone extension for Arquillian provides a simple way how to include functional tests for your application with a web-based user interface.
+Arquillian Drone brings the power of WebDriver into the Arquillian framework.
```

Comment:
I would rephrase this sentence. Maybe something like

> provides a simple way of writing functional tests for your ....

Maybe some additional bits about what Drone simplifies?